project/src/main/puzzle/pickups.gd

```diff
@@ -27,7 +28,10 @@ var _pickup_sfx_index := 0
 ## how many more pickup sounds should play after the current one
 var _remaining_pickup_sfx := 0
 
-onready var _puzzle_tile_map: PuzzleTileMap = get_node(_puzzle_tile_map_path)
+onready var _puzzle_tile_map: PuzzleTileMap = get_node(puzzle_tile_map_path)
```

Comment:
This should not be assigned onready, because piece_manager_path can (and will) be null in the editor.

Reply:
You're confusing PuzzleTileMap for PieceManagerPath; `puzzle_tile_map_path` is always assigned in the editor.